layersvt/screenshot_layer.md

```diff
@@ -57,3 +49,6 @@ Vulkan Cube demo from [Khronos/Vulkan-Tools](https://github.com/KhronosGroup/Vul
 adb shell pm grant com.example.VkCube android.permission.READ_EXTERNAL_STORAGE
 adb shell pm grant com.example.VkCube android.permission.WRITE_EXTERNAL_STORAGE
 ```
+## Layer Options
+
+The options for this layer are specified in VK_LAYER_LUNARG_screenshot.json. The option details are in [screenshot_layer.html](https://vulkan.lunarg.com/doc/sdk/latest/windows/screenshot_layer.html#user-content-layer-details).
```

Comment:
Change "The option details are in [screenshot_layer.html]" to

"The layer option details are documented in the [screenshot layer documentation]"